src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
 // export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  return (
+    <div>
+      <p className="Person__name">My name is {person.name}</p>
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+      <p className="Person__partner">
+        {person.isMarried
+          ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+          : 'I am not married'}
```

Comment:
Consider adding a check to ensure `person.partnerName` is defined before using it. This will prevent potential errors if `partnerName` is undefined. You can use a conditional check similar to how `person.age` is handled.